network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -390,6 +395,21 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        # In an effort to keep `product_page.html` changed as little as possible while there could be
+        # other work in other branches on that template, we're assigning ['product'] = self.
+        # TODO: At some point in the future, let's remove this and use `page.field_name` instead of
+        # using `product.field_name` in the template
+        context['product'] = self
+        context['categories'] = BuyersGuideProductCategory.objects.all()
+        context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
+        context['coralTalkServerUrl'] = settings.CORAL_TALK_SERVER_URL
+        context['pageTitle'] = pgettext(
+                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+                '*privacy not included') + f' - {self.title}'
+        return context
```

Comment:
We had this in the original django view:

```
    criteria = [
        'uses_encryption',
        'security_updates',
        'strong_password',
        'manage_vulnerabilities',
        'privacy_policy',
    ]
    total_score = 0
    num_criteria = len(criteria)

    # Calculate the minimum security score
    for i in range(num_criteria):
        value = product_dict[criteria[i]]
        if value == 'Yes':
            total_score += 1
        if value == 'NA':
            total_score += 0.5

..
    return render(request, 'product_page.html', {
        ...
        'security_score': total_score,
        'full_security_score': num_criteria
    })
```
But `security_score` and `full_security_score` wasn't being used in any templates or any other logic as far as I could tell, so I opted to keep it out of the Model-to-Page migration. If this was the wrong move I can add this back in pretty easily (Im not deleting an of the "old" code yet; reference: https://github.com/mozilla/foundation.mozilla.org/blob/pni-v4-2020/network-api/networkapi/buyersguide/views.py#L112-L148)